ui/src/pageLayout/components/UserWidget.tsx

```diff
@@ -71,8 +76,43 @@ const UserWidget: FC<Props> = ({
             />
           )}
         />
+        <FeatureFlag name="multiUser" equals={true}>
+          <TreeNav.UserItem
+            id="users"
+            label="Users"
+            linkElement={className => (
+              <a
+                className={className}
+                href={`${CLOUD_URL}/organizations/${org.id}${CLOUD_USERS_PATH}`}
```

Comment:
what's up with `/orgs/` and `/organizations/`?

Reply:
This is the route we ended up within Quartz, there's discussions ongoing around syncing this and users/members but that's outside the scope of this PR.